src/App.tsx

```diff
@@ -1,24 +1,219 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import {
+  FC,
+  useState,
+  useEffect,
+  useCallback,
+  useMemo,
+} from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import {
+  createTodo,
+  getTodos,
+  removeTodo,
+  patchTodo,
+} from './api/todos';
+import { NewTodoData, Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import { FilterType } from './enum/FilterType';
+import { USER_ID } from './utils/fetchClient';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoStatus, setTodoStatus] = useState<FilterType>(FilterType.All);
+  const [hasErrorMessage, setHasErrorMessage] = useState(false);
+  const [isLoadErrorMessage, setIsLoadErrorMessage] = useState(false);
+  const [isAddErrorMessage, setIsAddErrorMessage] = useState(false);
+  const [isDeleteErrorMessage, setIsDeleteErrorMessage] = useState(false);
+  const [isChangeStatusMessage, setIsChangeStatusMessage] = useState(false);
+  const [isAllStatusErrorMessage, setIsAllStatusErrorMessage] = useState(false);
```

Comment:
just combine your errors
```suggestion
  const [error, setError] = useState(Error.None);
```